dom/src/main/scala/org/scalajs/dom/ReadableStreamReader.scala

```diff
@@ -31,7 +31,7 @@ class ReadableStreamReader[+T](stream: ReadableStream[T]) extends js.Object {
     * If the reader is active, the cancel method behaves the same as that for the associated stream. When done, it
     * automatically releases the lock.
     */
-  def cancel[U](reason: U): js.Promise[U] = js.native
+  def cancel[U](reason: U): js.Promise[Unit] = js.native
```

Comment:
We no longer need the type parameter :)

```suggestion
  def cancel(reason: Any): js.Promise[Unit] = js.native
```

Reply:
Ah, committing the suggestion will not be enough, need to regenerate `api-reports` 😞

Reply:
Sorry about that :( we'd like to phase out the API reports b/c it's frustrating for contributors, we have a plan in https://github.com/scala-js/scala-js-dom/issues/503.